### DISCUSSION TITLE
Fix for Keywords Field in @project-sunbird/common-form-elements-full

### DISCUSSION THREAD

**Discussion by rajnishdargan** [Sunbird-Ed/Community#758] [Contribution]
While using the npm package [@project-sunbird/common-form-elements-full](https://www.npmjs.com/package/@project-sunbird/common-form-elements-full) version ^8.0.0, I noticed that the keywords field was not displaying in the form.

**Investigation and Solution:**

- When I installed the package, I discovered that it required `ngx-chips` as a dependency. However, the package documentation did not specify which version of `ngx-chips` to use.
- Initially, I installed version `2.2.2` of `ngx-chips`, which resolved the errors and allowed the application to run. However, the keywords field still did not display.
- Upon upgrading `ngx-chips` to version `3.0.0`, the keywords field began to display correctly.
- You can verify this issue and its resolution in this sample repository: https://github.com/rajnishdargan/test-sb-form.

**Steps Taken:**
To further analyze the issue, I set up the SunbirdEd-forms repository locally. During the process, I encountered some issues running the repo and made the following changes to make it functional:

Added `ngx-chips` as a peer dependency to resolve compatibility issues.
I have raised a [Pull Request](https://github.com/Sunbird-Ed/SunbirdEd-forms/pull/227) in the SunbirdEd-forms repository with these changes.

**Request:**
I kindly request the team to review the proposed changes in the PR. If everything looks good, please merge the changes and publish a new version of the package to ensure seamless integration for other users.

@pallakartheekreddy @HarishGangula
@coolbung @itsvick @snehal0904

**Comment by HarishGangula** [comment]
@rajnishdargan  There changes in the package name, can you check the PR once correct them.

**Reply by rajnishdargan** [reply]
Hi @HarishGangula 
I have updated the PR. Could you please review it?
[Pull Request Link](https://github.com/Sunbird-Ed/SunbirdEd-forms/pull/227)

Additionally, we need to update the [@project-sunbird/common-form-elements-full](https://www.npmjs.com/package/@project-sunbird/common-form-elements-full) package version. This should be done manually by building and publishing the new version, as it is a dependency for both the QuestionSet Editor and Course Editor.

[QuestionSet Editor Dependency](https://github.com/Sunbird-inQuiry/editor/blob/release-8.0.0/package.json#L32)
[Course Editor Dependency](https://github.com/Sunbird-Knowlg/sunbird-collection-editor/blob/release-6.1.0/package.json#L31)

**Reply by HarishGangula** [reply]
@rajnishdargan  Merged

**Reply by rajnishdargan** [reply]
Thanks @HarishGangula 
Could you please publish the new version of [@project-sunbird/common-form-elements-full](https://www.npmjs.com/package/@project-sunbird/common-form-elements-full) using the latest code.
